src/main/scala/files_metadata/infrastructure/FilesMetaPostgresRepository.scala

```diff
@@ -589,5 +589,66 @@ class FilesMetaPostgresRepository extends FilesMetaRepository {
     }
   }
 
-  override def deleteFileMeta( ownerUuid: UUID, uuid: UUID ): Unit = ???
+  override def deleteFileMeta( uuid: UUID ): Unit = {
+    val connection: Connection = pool.getConnection()
```

Comment:
The transaction is not needed since it is just executing one DELETE statement.

Reply:
Repaired:
![image](https://github.com/hawks-atlanta/metadata-scala/assets/47455237/166423a4-0656-489f-8055-cc21f9bb1690)
![image](https://github.com/hawks-atlanta/metadata-scala/assets/47455237/d5eecdf8-bba3-4899-80a2-03ad997f05af)